A2N Hymnal/ContentView.swift

```diff
@@ -39,12 +39,14 @@ struct ContentView: View {
         } //: NAVIGATION
         .sheet(isPresented: $showSettings, content: {
             SettingsView()
-            .environmentObject(viewModel)
-            .interactiveDismissDisabled(false)
+                .environmentObject(viewModel)
+                .interactiveDismissDisabled(false)
         })
+        //    DetailsView(hymn: Hymn)
```

Comment:
hi @jaypark wondering is there a reason the commented out "DetailsView(hymn: Hymn)" is added here?

Reply:
o oops, i’ll remove that.

Reply:
in the next PR.